00003_string_operations.js

```diff
@@ -0,0 +1,97 @@
+// 3. Enter 2 strings and do the following operations
+// concatenate / reverse / reverse concat / mix the letters / split in half / remove all white spaces / add a character after every specific letter /
+
+let string1;
+let string2;
+
+const concatinate = (string1, string2) => {
+    let concatinatedResult = string1.concat(string2);
+    return concatinatedResult
+}
+
+const reverseString = () => {
+    let reverseString1 = string1.split('').reverse().join('');
+    console.log("Reverse of the first string :", reverseString1);
+    let reverseString2 = string2.split('').reverse().join('');
+    console.log("Reverse of the second string :", reverseString2);
+    let reverseConcat = reverseString1.concat(reverseString2);
+    console.log("Reverse concatinated :", reverseConcat);
+}
+
+const stringMixUp = (string1, string2) => {
+    let stringMix = [];
+    if (string1.length > string2.length) {
+        for (let i = 0; i < string1.length; i++) {
+            if (i + 1 > string2.length) {
+                stringMix = stringMix + string1[i];
+            } else {
+                let currentStringValuesofString1AndString2 = string1[i].concat(string2[i]);
+                stringMix[i] = currentStringValuesofString1AndString2;
+            }
+
+        }
+    } else {
+        for (let i = 0; i < string2.length; i++) {
+            if (i + 1 > string1.length) {
+                stringMix = stringMix + string2[i];
+            } else {
+                let currentStringValuesofString1AndString2 = string1[i] + string2[i];
+                stringMix[i] = currentStringValuesofString1AndString2;
+            }
+
+        }
+
+    }
+    let stringMixResult = stringMix.replace(/,/g, "");
+    return stringMixResult
+}
+
+const splitedHalfOfString1 = (string1) => {
+    if (string1.length % 2 == 0) {
+        return string1.slice(0, string1.length / 2)
+    } else return "cannot be devided."
```

Comment:
????

Reply:
If the string length is odd, it couldn't be divided.